parachain/pallets/inbound-queue/src/mock.rs

```diff
@@ -221,6 +266,7 @@ impl inbound_queue::Config for Test {
 	type WeightToFee = IdentityFee<u128>;
 	type LengthToFee = IdentityFee<u128>;
 	type MaxMessageSize = ConstU32<1024>;
+	type AssetTransactor = SuccessfulTransactor;
```

Comment:
Would be nice to have some tests that checks the `BurnFees` code paths but not a blocker.

Reply:
[Change](https://github.com/Snowfork/polkadot-sdk/compare/89a79f518380c55a9dadf3098a025daadb37c08c...452ae5f8fb600b82d4267d519fa51708420bfdf2#diff-5ded4b43c234d6f2d43aff2b2b63adfde7a5ba2dd9fc591cd818335e20b6303b) to cover that.

Also, token-transfer-related smoke tests work as expected.